src/main/kotlin/uk/gov/justice/digital/assessments/services/EpisodeService.kt

```diff
@@ -107,31 +112,22 @@ class EpisodeService(
 
   private fun prepopulateFromSource(
     episode: AssessmentEpisodeEntity,
-    sourceName: String?,
-    questions: List<ExternalSourceQuestionSchemaDto>,
+    sourceName: String,
+    questionSchemas: List<ExternalSourceQuestionSchemaDto>,
     latestCompleteEpisodeEndDate: LocalDateTime?
   ) {
-    val questionsByExternalSourceEndpoint = questions.groupBy { it.externalSourceEndpoint }
-    episode.prepopulatedFromOASys = sourceName == OASYS_SOURCE_NAME
```

Comment:
episode.prepopulatedFromOASys = sourceName == OASYS_SOURCE_NAME needs to be added for RSR